src/Chapter2.hs

```diff
@@ -621,7 +642,9 @@ Write a function that takes elements of a list only on even positions.
 >>> takeEven [2, 1, 3, 5, 4]
 [2,3,4]
 -}
-takeEven = error "takeEven: Not implemented!"
+takeEven [] = []
```

Comment:
I would encourage to add top-level type signatures to all functions. It's highly recommended, especially when learning Haskell :slightly_smiling_face: Type signatures also help a bit with documentation and understanding, what is actually happening

Reply:
I forgot to add signature. I will add signature and pull request again.